src/app/accounts/add-earn.ts

```diff
@@ -33,6 +39,13 @@ export const addEarn = async ({
   if (validatedPhoneMetadata instanceof Error)
     return new InvalidPhoneMetadataForRewardError(validatedPhoneMetadata.name)
 
+  const userIP = await UsersIpRepository().findById(user.id)
```

Comment:
I think we use `userIp` instead of `userIP` in the rest of the codebase

Reply:
I pulled this from `src/app/users/get-user.ts`'s `updateUserIPsInfo` method, which uses the variable name of `userIP` in the same context.

Grepping around the code base, it looks like `userIP` is being used as variable naming in a caller, while `userIp` is used as the variable name in the callee's signature.

Happy to change, if desired. Just let me know how you would like to proceed.

Reply:
ok I thought we had some consistency on this but it seems we don't, so no need to change it